s3.tf

```diff
@@ -28,4 +28,34 @@ resource "aws_s3_bucket_versioning" "ecr_viewer" {
   versioning_configuration {
     status = "Enabled"
   }
-}
\ No newline at end of file
+}
+
+resource "aws_s3_bucket" "logging" {
+  bucket        = local.s3_logging_bucket_name
+  force_destroy = true
+  tags          = local.tags
+}
+
+resource "aws_s3_bucket_policy" "logging" {
+  bucket = aws_s3_bucket.logging.id
+  policy = data.aws_iam_policy_document.logging.json
+}
+
+resource "aws_s3_bucket_public_access_block" "logging" {
+  bucket                  = aws_s3_bucket.logging.id
+  block_public_acls       = true
+  block_public_policy     = true
+  ignore_public_acls      = true
+  restrict_public_buckets = true
+}
+
+# https://avd.aquasec.com/misconfig/aws/s3/avd-aws-0132/
+# trivy:ignore:AVD-AWS-0132
```

Comment:
I have a concern with ignoring this. The best practice would be to utilize a customer-managed key, per the example here: https://avd.aquasec.com/misconfig/aws/s3/avd-aws-0132/#Terraform Is there a compelling reason to not go with that approach?

Reply:
Nope, it's a worthwhile change and I'll push it. It likely does have downstream impacts that require permission updates.